### PR TITLE
Don't flush and migrate in generate_test_data

### PR DESCRIPTION
**Pull request by bfirsh**
This was causing weirdness because intitial data is created in
migrations, but sqlflush then deletes it, then migrate doesn't
create the data again because it thinks the database is up to
date.

### CODE REVIEW

File: core/management/commands/generate_test_data.py [hunk 1]
```diff
@@ -15,31 +15,9 @@
...
-            for statement in statements:
-                if not statement:
-                    continue
-                cursor.execute(statement)
-
-        call_command('migrate')
-
-        # setting the seed so output is consistent
-        fake = Faker()
-        fake.seed(1)
```
Comment (jessykate) [inline core/management/commands/generate_test_data.py]: do we still want these? i'm not sure if there was a reason @jonathan-s designed this to have consistent output. having non-consistent output would test more cases, just not deterministically.

Comment (bfirsh) [reply core/management/commands/generate_test_data.py]: Yes - oops. It's back in a rebased version.

**Comment by jonathan-s** [comment]
> This was causing weirdness because intitial data is created in
migrations, but sqlflush then deletes it, then migrate doesn't
create the data again because it thinks the database is up to
date.

@bfirsh What kind of weirdness does it create? I'd just be keen to checkout the weirdness firsthand. So I know what to look for.

**Comment by bfirsh** [comment]
Just a couple of weird states that can’t be get into in any other way. 

Primary weirdness:
- Migrations are run manually. Some initial data is created (currencies)
- sqlflush wipes all tables _except the migration state_
- generate_test_data migrates again, thinks it is up to date, so initial data migrations are not run again

Secondary weirdness: tests don’t use migrations, but the generate test data command runs them. 

Above all, just seems weird that this command wipes the database. Seems out of its remit - perhaps we could have another command to do that if we want to do that.

**Pull request merged**